supporting_content/dimensions/listunitprice.md

```diff
@@ -0,0 +1,83 @@
+# Column: ListUnitPrice
+
+## Example provider mappings
+
+Current column mappings found in available data sets:
+
+| Provider  | Data set                     | Column                   |
+|:----------|:-----------------------------|:-------------------------|
+| AWS       | CUR;<br>AWS Price List API   | pricing/publicOnDemandCost: The total cost for the line item based on public On-Demand Instance rates. If you have SKUs with multiple On-Demand public costs, the equivalent cost for the highest tier is displayed. For example, services offering free-tiers or tiered pricing.<br>Since the pricing/publicOnDemandCost currently doesn't provide the correct price in all tier-based pricing use cases, we need to rely on the AWS Price List API. |
```

Comment:
Correction required: The reference should be to 'pricing/publicOnDemandRate' instead of 'pricing/publicOnDemandCost'.